### DISCUSSION TITLE
Downtime duration in notification template

### DISCUSSION THREAD

**Discussion by mdragnes** [grafana/grafana#60079] [Alerting]
Would like to request a feature to include duration downtime in a resolved alert notification.
We send out resolved alert notifications as emails on state changes, and would be helpful to get this information in a notification template to include in the sent email.

We know that the database has current_state_end and current_state_since, and we use transform operator on these two fields to calculate the duration.

Could not find any similar topics about this request specifically, but please let me know if such a solution is easily available.

Example: 
=============================================================================

Sensor: Ping (Ping) ***
[Link to Grafana dashboard]

New Status: Down ended (now: Up) (Downtime: 120 s)

=============================================================================
Date/Time: 12/8/2022 8:46:58 PM (UTC)
Last Result: 856 msec (Ping Time)
Last Message: OK

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
